src/components/Dashboard/FilingHistoryList/FilingTemplate.vue

```diff
@@ -63,7 +63,7 @@
       <slot name="body">
         <!-- is this a generic paid (not yet completed) filing? -->
         <div
-          v-if="isStatusPaid"
+          v-if="isStatusPaid || isStatusApproved"
```

Comment:
Approved looks like Paid.

(Rejected is handled in ContinuationIn component, not here in the template.)